data/102/1001/en_US.json

```diff
@@ -1,13 +1,13 @@
-{
-	"102": {
-		"1001": {
-			"name": "ACCOUNT_NOT_FOUND",
-			"message": "Unable to find the specified user.",
-			"short_description": "Unknown cause.",
-			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
-			"long_solution": "Unknown solution.",
-			"support_link": "https://preten.do/102-1001"
-		}
-	}
+{
+	"102": {
+		"1001": {
+			"name": "ACCOUNT_NOT_FOUND",
+			"message": "Unable to find the specified user.",
+			"short_description": "The Nintendo/Pretendo Network ID entered was incorrect or does not exist.",

```

Comment:
If a short description is being added, then a long one should also be added. The `short` fields are designed for use by the Discord bot. The `long` fields are used for the website. Only having one of them makes the data unavailable by the other.

You can see a good example of how to format these fields here https://github.com/PretendoNetwork/error-codes/blob/master/data/104/2210/en_US.json

This goes for all cases of this, as there are multiple in this PR